### PR TITLE
feat: validate status field in order

### PR DESCRIPTION
**Pull request by DPDeepak**
Ticket: https://github.com/nodexpertsdev/RnD/issues/126

### CODE REVIEW

File: expressEngine/model/lib/index.js [hunk 1]
```diff
@@ -3,13 +3,15 @@ class ModelLib {
     this.roleEnum = ['system-admin', 'supplier'];
   }
 
-  validateEmail = email => {
-    var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-    return re.test(String(email).toLowerCase());
+  orderStatus =() => ['confirmed', 'shipped', 'In-Delivery', 'delivered', 'closed'];
```
Comment (aman-successive) [inline expressEngine/model/lib/index.js]: add cancelled field instead of closed

Comment (DPDeepak) [reply expressEngine/model/lib/index.js]: 'closed' is mention in the issue. please have a look.

Comment (chiragSuccessive) [reply expressEngine/model/lib/index.js]: @aman-successive I think Cancel field is for order items, order is always closed.

Comment (mayank-nx) [inline expressEngine/model/lib/index.js]: @DPDeepak use camelCase for status.

**Pull request merged**